spec/helpers/application_helper_spec.rb

```diff
@@ -25,4 +25,27 @@
       end
     end
   end
+
+  describe "title" do
+
+    context "nil input" do
+      it 'returns nil' do
+        expect(title).to be nil
+      end
+    end
+
+    context "non-nil input" do
+      it 'returns nil for an empty string' do
+        expect(title('')).to be nil
+      end
+
+      it 'returns the title for a non-empty string' do
+        helper.title('Courses')
+        expect(helper.instance_variable_get(:@view_flow).
```

Comment:
Oh sorry, I thought I edited that part. if `:@view_flow` ever changes in a future rails version is what I meant.